client/src/model/dashboards/widget-props/current-values-props.ts

```diff
@@ -0,0 +1,67 @@
+import { CurrentValuesWidgetProps } from '../../../widgets/current-values-widget';
+
+export const fakeProps: CurrentValuesWidgetProps = {
+	connections: [
```

Comment:
Could you please exchange the channel names with the new, "real" channel names in the mock server?

(I changed the channel names within `main` during work on this branch)

Reply:
Here we go.
Done in https://github.com/wuespace/telestion-project-daedalus2/pull/31/commits/a975cf6a1b015e6eaa108d81e6d046f8a97b9764